src/e2e-test/java/io/cdap/plugin/gcs/stepsdesign/GCSSource.java

```diff
@@ -128,4 +131,21 @@ public void selectGCSSourcePropertyReadFileRecursiveAs(String value) {
   public void selectGCSSourcePropertyPathFilenameOnlyAs(String value) {
     CdfGcsActions.selectPathFilenameOnly(value);
   }
+
+
+  @Then("Read Credentials file {string} with value: {string}")
+  public void readCredentialsFileWithValue(String pluginProperty, String keys) throws IOException {
+    FileReader file = new FileReader("C:/Users/ishak/Downloads/google-cloud/src/e2e-test/resources/credentials");
```

Comment:
Don't hardcode the value here in the parameter, instead to get it from plugin parameter.

Reply:
done
commit id- https://github.com/cloudsufi/google-cloud/commit/defc54df110255ef539dea8d206481cc145b2fec